scripts/plugins/storage/sqlite.sh

```diff
@@ -344,7 +340,7 @@ case "$engine_management" in
 
         # Unexpected value from the configuration file
         sqlite_log "err" "FogLAMP cannot start." "all" "pretty"
-        sqlite_log "err" "Missing plugin information in FogLAMP configuration file foglamp.json" "all" "pretty"
+        sqlite_log "err" "Missing plugin information the storage microservice" "all" "pretty"
```

Comment:
add "from":

Missing plugin information the storage microservice

===>

Missing plugin information from the storage microservice

Reply:
Fixed - thanks!